chart/templates/_helpers.tpl

```diff
@@ -5,7 +5,7 @@
 
 {{- define "connector.host" -}}
```

Comment:
Thinking about it...this block can be simplified further. We no longer need the `if...else`; just the `printf` inside the `if` part.